docs/endpoints.rst

```diff
@@ -1021,9 +1021,227 @@ Show number of users editing buildings before, during and after Nepal earthquake
+
+**Example request**:
+
+Number of contributions to the building 'Stadthalle Heidelberg' between 2010 and 2020.
+
+  .. tabs::
+
+    .. code-tab:: bash curl (GET)
+
+       curl -X GET 'https://api.ohsome.org/v1/contributions/count?bboxes=8.699053,49.411842,8.701311,49.412893&filter=id:way/140112810&time=2010-01-01,2020-01-01'
+
+    .. code-tab:: bash curl (POST)
+
+       curl -X POST 'https://api.ohsome.org/v1/contributions/count' --data-urlencode 'bboxes=8.699053,49.411842,8.701311,49.412893' --data-urlencode 'time=2010-01-01,2020-01-01' --data-urlencode 'filter=id:way/140112810'
+
+    .. code-tab:: python Python
+
+        import requests
+        URL = 'https://api.ohsome.org/v1/contributions/count'
+        data = {"bboxes": "8.699053,49.411842,8.701311,49.412893", "time": "2010-01-01,2020-01-01", "filter": "id:way/140112810"}
+        response = requests.post(URL, data=data)
+        print(response.json())
+
+    .. code-tab:: r R
+
+        library(httr)
+        r <- POST("https://api.ohsome.org/v1/contributions/count", encode = "form", body = list(bboxes = "8.699053,49.411842,8.701311,49.412893", time = "2010-01-01,2020-01-01", filter = "id:way/140112810"))
+        r
+
+
+**Example response**:
+
+  .. tabs::
+
+   .. code-tab:: json curl (GET)
+
+	{
+	  "attribution":{
+	    "url":"https://ohsome.org/copyrights",
+	    "text":"© OpenStreetMap contributors"
+	  },
+	  "apiVersion":"1.4.0",
+	  "result":[
+	    {
+	      "fromTimestamp":"2010-01-01T00:00:00Z",
+	      "toTimestamp":"2020-01-01T00:00:00Z",
+	      "value":15.0
+	    }
+	  ]
+	}
+
+   .. code-tab:: json curl (POST)
+
+	{
+	  "attribution":{
+	    "url":"https://ohsome.org/copyrights",
+	    "text":"© OpenStreetMap contributors"
+	  },
+	  "apiVersion":"1.4.0",
+	  "result":[
+	    {
+	      "fromTimestamp":"2010-01-01T00:00:00Z",
+	      "toTimestamp":"2020-01-01T00:00:00Z",
+	      "value":15.0
+	    }
+	  ]
+	}
+
+
+   .. code-tab:: json Python
+
+	{
+	  "attribution":{
+	    "url":"https://ohsome.org/copyrights",
+	    "text":"© OpenStreetMap contributors"
+	  },
+	  "apiVersion":"1.4.0",
+	  "result":[
+	    {
+	      "fromTimestamp":"2010-01-01T00:00:00Z",
+	      "toTimestamp":"2020-01-01T00:00:00Z",
+	      "value":15.0
+	    }
+	  ]
+	}
+
+   .. code-tab:: json R
+
+	{
+	  "attribution":{
+	    "url":"https://ohsome.org/copyrights",
+	    "text":"© OpenStreetMap contributors"
+	  },
+	  "apiVersion":"1.4.0",
+	  "result":[
+	    {
+	      "fromTimestamp":"2010-01-01T00:00:00Z",
+	      "toTimestamp":"2020-01-01T00:00:00Z",
+	      "value":15.0
+	    }
+	  ]
+	}
 
+
+.. http:post :: /contributions/count/density
+
+   Get the density of the count of contributions divided by the total area in square-kilometers. This endpoint does not support the deprecated ``types``, ``keys``, ``values`` parameters.
```

Comment:
```suggestion
   Get the density of the count of contributions in the total query area in counts per square-kilometers. This endpoint does not support the deprecated ``types``, ``keys``, ``values`` parameters.
```

Reply:
This wording would then not fit to the wording used in the other density description in line 134.

Reply:
but this wording is the correct one :stuck_out_tongue: becuase a density of something divided by the area would have the unit of `thing / length^4` (instead of `thing / lenght^2`)

Reply:
Yeah I get now that you can also understand the old wording in a different way.. Then just fix also the old wording in the other line and it's fine :)